python/ray/tests/test_response_cache.py

```diff
@@ -0,0 +1,158 @@
+from ray.util.client.common import (_id_is_newer, ResponseCache,
+                                    OrderedResponseCache, INT32_MAX)
+import threading
+import time
+
+import pytest
+
+
+def test_id_is_newer():
```

Comment:
Is it possible to add a high level description similar to all the tests in this file? A great example of these helpful description are in test_client_reconnect.py which you wrote above!